.github/workflows/release.yml

```diff
@@ -6,35 +6,11 @@ on:
       - master
 
 jobs:
-  release-please:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: GoogleCloudPlatform/release-please-action@v2
-        id: release
-        with:
-          token: ${{ secrets.BOT_TOKEN }}
-          release-type: node
-          package-name: 'ml-array-xy'
-          bump-minor-pre-major: Yes
-      - uses: actions/checkout@v2
-        # These if statements ensure that a publication only occurs when a new release is created
-        if: ${{ steps.release.outputs.release_created }}
-        with:
-          fetch-depth: 0
-      - uses: actions/setup-node@v2
-        with:
-          node-version: 14.x
-          registry-url: 'https://registry.npmjs.org'
-        if: ${{ steps.release.outputs.release_created }}
-      - run: npm install
-        if: ${{ steps.release.outputs.release_created }}
-      - run: npm run bootstrap
-        if: ${{ steps.release.outputs.release_created }}
-      - run: git config --global user.name "cheminfo-bot"
-        if: ${{ steps.release.outputs.release_created }}
-      - run: git config --global user.email "10880445+cheminfo-bot@users.noreply.github.com"
-        if: ${{ steps.release.outputs.release_created }}
-      - run: npm run publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_BOT_TOKEN }}
-        if: ${{ steps.release.outputs.release_created }}
+  release:
```

Comment:
sorry, we should not change this workflow. it has different steps because this is a monorepo